src/dune_pkg/lock_dir.mli

```diff
@@ -50,8 +50,12 @@ type t =
   ; packages : Pkg.t Package_name.Map.t
   }
 
+val create_latest_version : Pkg.t Package_name.Map.t -> t
+
 val path : Path.Source.t
 
 val metadata : Filename.t
 
 module Metadata : Dune_sexp.Versioned_file.S with type data := unit
+
+val instantiate_on_disk : lock_dir_path:Path.Source.t -> t -> unit
```

Comment:
How about `write_disk`? `instantiate` sounds like we're doing more than just some serialization.